index.html

```diff
@@ -9150,6 +9165,51 @@ <h2 id="AudioWorkletNodeOptions">
                 </dd>
               </dl>
             </section>
+            <section>
+              <h3>
+                Configuring Channels with <a>AudioWorkletNodeOptions</a>
+              </h3>
+              <p>
+                With a combination of <a data-link-for=
+                "AudioWorkletNodeOptions">numberOfInputs</a>, <a data-link-for=
+                "AudioWorkletNodeOptions">numberOfOutputs</a> and
+                <a data-link-for=
+                "AudioWorkletNodeOptions">outputChannelCount</a>, various
+                channel configurations can be achieved.
```

Comment:
Can we use a table somehow?

Reply:
Hmm. I actually prefer this way. I think a list is better than using `<table>` element. We only have 3 cases with a single description for each.

Reply:
Ok